src/Adapter/EntityTranslation/EntityTranslator.php

```diff
@@ -62,11 +62,6 @@ class EntityTranslator implements EntityTranslatorInterface
      */
     protected $tableName;
 
-    /**
-     * @var int
-     */
-    protected $shopId;
```

Comment:
Please describe the BC Break

Reply:
It's not a BC, it has been introduced in 1.7.8.x